src/components/game/container.js

```diff
@@ -1,20 +1,26 @@
 import { connect } from 'react-redux';
 
-import { selectSquare, goToMove, changeMovesOrder } from '../../actions';
+import { selectSquare, goToMove } from '../../reducers/state-of-game/actions';
 
+import { gameState } from './controller';
 import Game from './component';
 
-const mapStateToProps = ({ history, moveNumber, xIsNext }) => ({
-  squares: history[moveNumber].squares,
-  history,
-  moveNumber,
-  xIsNext,
+const mapStateToProps = (history) => ({
+  history: history.app.status.history,
+  moveNumber: history.app.status.moveNumber,
+  xIsNext: history.app.status.xIsNext,
+  state: history.app.status,
 });
 
 const mapDispatchToProps = (dispatch) => ({
-  onSelectSquare: (id) => dispatch(selectSquare(id)),
+  onSelectSquare: (squareId, xIsNext) => {
+    const squaresPlayed = gameState(mapStateToProps.history);
```

Comment:
```gameState(mapStateToProps.history);``` 
What does this piece do?

Reply:
I was trying to pass the history to the function but it wasn't working so removed it